### ISSUE TITLE
How to Toggle: Window with Frame -> Fullscreen no Frame

### ISSUE THREAD

**Issue by marcgfx** [nwjs/nw.js#7280]
I've just updated from 33.3 to 43.1 so quite a big step. Main reason was because the registered keys stopped working (I'm using F11 to toggle fullscreen/windowed) after reloading the app. The issue I am now facing is a bit different.

How do I get the windowed mode to have a frame and the fullscreen mode to have no frame? I currently has a white frame in windowed mode and a black one in fullscreen.

"window": {
	"toolbar": true,
	"frame":true,
	"fullscreen":false,
	"width":1024,
	"height":768
}

I am using 
nw.Window.get().leaveFullscreen();
nw.Window.get().enterFullscreen();
to toggle the two modes.

It seems I need a way to toggle the frame as well.

**Comment by rogerwang** [comment]
Does `--disable-features=nw2` make any difference there?

**Comment by marcgfx** [comment]
yes it actually solves the problem. back to the way it was before.
